ledfx/devices/wled.py

```diff
@@ -40,36 +40,36 @@ class WLEDDevice(NetworkedDevice):
         "E131": E131Device,
     }
 
-    DEVICE_CONFIGS = {
-        "UDP": {
-            "name": None,
-            "ip_address": None,
-            "pixel_count": None,
-            "port": 21324,
-            "udp_packet_type": "DNRGB",
-            "timeout": 1,
-            "minimise_traffic": True,
-        },
-        "DDP": {
-            "name": None,
-            "ip_address": None,
-            "pixel_count": None,
-        },
-        "E131": {
-            "name": None,
-            "ip_address": None,
-            "pixel_count": None,
-            "universe": 1,
-            "universe_size": 510,
-            "channel_offset": 0,
-            "packet_priority": 100,
-        },
-    }
-
     def __init__(self, ledfx, config):
         super().__init__(ledfx, config)
         self.subdevice = None
 
+        self.DEVICE_CONFIGS = {
```

Comment:
Would appreciate comment if I should be using deepcopy instead, or if the name should become lower case.

Reply:
Hmm I don't think we need to copy it, but since its no longer a global variable should we make it lower case?

Reply:
done